Pod/Classes/KeyboardAdjuster.swift

```diff
@@ -127,13 +123,26 @@ extension KeyboardAdjuster where Self: UIViewController {
      - copyright: ©2016 Lionheart Software LLC
      - date: February 18, 2016
      */
-    public func activateKeyboardAdjuster(_ showBlock: AnyObject?, hideBlock: AnyObject?) {
+    public func activateKeyboardAdjuster(_ showBlock: (()->())?, hideBlock: (()->())?) {
         // Activate the bottom constraint.
         keyboardAdjusterConstraint?.isActive = true
 
         let notificationCenter = NotificationCenter.default
-        notificationCenter.addObserver(self, selector: #selector(keyboardWillHide(_:)), name: NSNotification.Name.UIKeyboardWillHide, object: hideBlock)
-        notificationCenter.addObserver(self, selector: #selector(keyboardWillShow(_:)), name: NSNotification.Name.UIKeyboardWillShow, object: showBlock)
```

Comment:
The `object` argument here is for the following purpose:

> The object whose notifications the observer wants to receive; that is, only notifications sent by this sender are delivered to the observer.

This means that only `UIKeyboardWillShow` notifications sent by the `showBlock` object will be received. This doesn't really make sense and breaks the KeyboardAdjuster functionality when `showBlock` is not `nil`.